src/flutter/templates.ts

```diff
@@ -0,0 +1,90 @@
+import { makeCodeSnippet } from '../utils/clack-utils';
+
+export const sentryImport = `import 'package:sentry_flutter/sentry_flutter.dart';\n`;
+
+export function pubspecOptions(project: string, org: string): string {
+  return `sentry:
+  upload_debug_symbols: true
+  upload_source_maps: true
+  project: ${project}
+  org: ${org}
+`;
+}
+
+export function sentryProperties(authToken: string): string {
+  return `auth_token=${authToken}`;
+}
+
+export function initSnippet(
+  dsn: string,
+  selectedFeaturesMap: {
+    tracing: boolean;
+    profiling: boolean;
+  },
+  runApp: string,
+): string {
+  let snippet = `await SentryFlutter.init(
+    (options) {
+      options.dsn = '${dsn}';`;
+
+  if (selectedFeaturesMap.tracing) {
+    snippet += `
+      // Set tracesSampleRate to 1.0 to capture 100% of transactions for tracing.
+      // We recommend adjusting this value in production.
+      options.tracesSampleRate = 1.0;`;
+  }
+
+  if (selectedFeaturesMap.profiling && selectedFeaturesMap.tracing) {
+    snippet += `
+      // The sampling rate for profiling is relative to tracesSampleRate
+      // Setting to 1.0 will profile 100% of sampled transactions:
+      options.profilesSampleRate = 1.0;`;
+  }
+
+  snippet += `
+    },
+    appRunner: () => runApp(${runApp}),
```

Comment:
imo we should wrap with `SentryWidget` out of the box because a couple important features rely on it, replay, screenshot, user interactions etc...